website/content/commands/connect/ca.mdx

```diff
@@ -42,8 +42,18 @@ Subcommands:
 
 This command displays the current CA configuration.
 
+The table below shows this command's [required ACLs](/api#authentication). Configuration of
+[blocking queries](/api/features/blocking) and [agent caching](/api/features/caching)
+are not supported from commands, but may be from the corresponding HTTP endpoint.
+
+| ACL Required     |
+| ---------------- |
+| `operator:write` |
+
 Usage: `consul connect ca get-config [options]`
 
+Corresponding HTTP API Endpoint: [\[GET\] /v1/connect/ca/configuration](https://www.consul.io/api-docs/connect/ca#get-ca-configuration)
```

Comment:
Ah, I see. There is no separate page for this "command" because it's a "subcommand" so you had to move this under the usage. Makes sense.